### PR TITLE
MIG-421 Using unified logger in MTC

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-421

CP to 4.5, 4.6, 4.7.

### CODE REVIEW

File: modules/migration-using-mig-log-reader.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+.Procedure
+
+* Run the following command to display a single migration log:
++
+[source,terminal]
+----
+$ oc logs -f --selector logreader=mig -n openshift-migration -c color <1>
+----
+<1> You can use the `-c plain` option to display the log without colors.
```
Comment (hu-weihua) [inline modules/migration-using-mig-log-reader.adoc:20]: Not sure why, when use `View file` option in `...`, line 20 is shown as 
```
  i. You can use the -c plain option to display the log without colors.
```
I suppose the expected format should be below line,right?
```
   1. You can use the -c plain option to display the log without colors.
```
Could help to double check the document syntax?

Comment (hu-weihua) [reply modules/migration-using-mig-log-reader.adoc:20]: Do we just primarily recommend customer to use `mig-log-reader` with `-f` option, to view logs in stream? 

Or it is better to explain `-c color` `-c plain` and `-f` one by one, customer will decide how to use them.

Customer maybe have two kinds of requirement,
* monitor the current migration process logs in stream
* look back at the past logs

Comment (apinnick) [reply modules/migration-using-mig-log-reader.adoc:20]: > Not sure why, when use `View file` option in `...` to review, line 20 is shown as
> 
> ```
>   i. You can use the -c plain option to display the log without colors.
> ```
> 
> I suppose the expected format should be below line,right?
> 
> ```
>    1. You can use the -c plain option to display the log without colors.
> ```
> 
> Could help to double check the document syntax?

This is for a callout. See the doc preview. The syntax is correct.

Comment (apinnick) [reply modules/migration-using-mig-log-reader.adoc:20]: > Do we just primarily recommend customer to use `mig-log-reader` with `-f` option, to view logs in stream?
> 
> Or it is better to explain `-c color` `-c plain` and `-f` one by one, customer will decide how to use them.

I do not think we need to explain the options separately. They are obvious and the customer will know how to use them.

> 
> Customer maybe have two kinds of requirements,
> 
>     * monitor the current migration process logs in stream

I understand that the `mig-log-reader` is for tailing current logs.

>     * look back at the past logs

If the customer needs to look at past logs, that is explained in another section: https://docs.openshift.com/container-platform/4.6/migration/migrating_3_4/troubleshooting-3-4.html#migration-downloading-logs_migrating-3-4

**Comment by hu-weihua** [comment]
@apinnick thanks for your quick reply.
LGTM. thanks a lot.

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Pull request merged**